packages/bpk-component-split-input/src/BpkInputField.js

```diff
@@ -0,0 +1,70 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import React, { Component } from 'react';
+import BpkInput from 'bpk-component-input';
+import { cssModules } from 'bpk-react-utils';
+import PropTypes from 'prop-types';
+
+import STYLES from './BpkInputField.module.scss';
+
+const getClassName = cssModules(STYLES);
```

Comment:
Like Ollie said, would be great to rename this to `getClassName` just to follow the style in the other bpk components 🙂